sql/hive/src/main/scala/org/apache/spark/sql/hive/HiveMetastoreCatalog.scala

```diff
@@ -286,3 +347,33 @@ private[hive] class HiveMetastoreCatalog(sparkSession: SparkSession) extends Log
     }
   }
 }
+
+private[hive] object HiveMetastoreCatalog {
```

Comment:
nit: don't add `private[xxx]` if this class is already in "private" package. `org.apache.spark.sql.hive` is private.